spinalcordtoolbox/deepseg/models.py

```diff
@@ -321,6 +331,17 @@
                              'the University of Zurich.',
          'url': 'https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1',
          'models': ['model_seg_gm_wm_mouse_nnunet']},
+    'seg_sc_ms_lesion_stir_psir':
+        {'description': 'Segmentation of spinal cord and MS lesions for STIR and inverted PSIR (multiplied by -1) contrasts ',
```

Comment:
Minor, minor nitpick, but the "description" is a bit longer than the other ones. On a fresh terminal (80 pixels wide), the description wraps quite a lot:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/db4e4e7b-d81d-41fb-8a0e-113ea0d2166f)

So, I would suggest a slight rewording just to make it fit:

```suggestion
        {'description': 'MS lesion/SC seg for STIR/inverted PSIR contrasts',
```

I think the extended description does a good job already of explaining the "multiplied by 1" part?

Reply:
i would not mention the multiplied by -1 in the description of the task

Reply:
To clarify: I thought that the code already multiplies by -1 if the string 'psir' is identified-- chatting with @plbenveniste I understand this is not the case, and the user is expected to do it. Problem: they won't understand/ will make mistake, etc.

bottomline: we should do it ourselves, probably in the wrapper (good idea?) or somewhere else...

Reply:
Yes I think that's a good idea. 
It just requires to do : `sct_maths -i IMAGE -mul -1 -o OUTPUT`

Should the user then specify in the command if the image is PSIR or STIR ? Or should the code look for "PSIR" in the file name ?

Reply:
i though there was already a dedicated use of the flag ‘-c stir/psir’ (otherwise there is no use for this flag since it’s the same model for both contrasts)

Reply:
> otherwise there is no use for this flag since it’s the same model for both contrasts

That is exactly the reason I opened the issue 

- #4445

Reply:
I have some ideas for how we can update the `-c` argument to support switching between PSIR/STIR contrasts, without an error occurring if `-c` is not passed.

I will try to get to this tonight, or early Monday at the latest.

Reply:
Nice ! But I guess that we always want `-c` to be passed for the model, to avoid cases' where people forget to multiply PSIR images by -1. 
However, I agree with you that we don't want `-c` to be required for the majority of `sct_deepseg` models

Reply:
> Nice ! But I guess that we always want -c to be passed for the model, to avoid cases' where people forget to multiply PSIR images by -1.

The model won't be used by a lot of people in the next few months, so I think it's OK to not enforce this flag. At some point we will likely have a model that won't require to multiply by -1, so this is a temporary hack anyway

Reply:
I chose to enforce `-c` flag, but only for `-task seg_sc_ms_lesion_stir_psir`. So, this behavior will still be invisible to 99% of users, but for people who are using `seg_sc_ms_lesion_stir_psir`, they will explicitly choose whether to invert or not.